arbitrum-docs/launch-orbit-chain/how-tos/orbit-chain-finality.md

```diff
@@ -0,0 +1,29 @@
+# Orbit Chain Confirmation and Finality
+
+## Child Chain Transactions
+
+Generally, transactions executed through the sequencer on Orbit chains [achieve finality](/tx-lifecycle) equivalent to their parent chain once the relevant transaction data has been [posted in a batch](/sequencer). This means that transactions on Orbit chains are considered final in minutes.
+
+## Parent Chain → Child Chain Transactions
+
+Messages being sent through the delayed inbox of a parent chain as [retryable tickets](/arbos/l1-to-l2-messaging#retryable-tickets), including deposits through token bridges, are released by a [well-behaved Sequencer](/inside-arbitrum-nitro/#if-the-sequencer-is-well-behaved) once they have good confidence of finality on a parent chain. On an L2 settling to Ethereum, this means that a sequencer will release delayed messages to the inbox after 40 blocks. Following this, the transaction will need to complete another finality period for the Ethereum transaction that promoted them to achieve finality.
+
+Orbit L3s may configure the finality of transactions executed through the delayed inbox to depend on different layers of finality. By default, Orbit chains will rely on the number of L1 block confirmations, effectively finalizing an L3 deposit as soon as L1 finalizes the batch posted by Arbitrum One or when a DACert is posted by Arbitrum Nova. This would be on the order of tens of minutes.
```

Comment:
```suggestion
Orbit L3s may configure the finality of transactions executed through the delayed inbox to depend on different layers of finality. By default, Orbit chains will rely on the number of L1 block confirmations, effectively finalizing an L3 deposit as soon as L1 finalizes the batch posted by <a data-quicklook-from="arbitrum-one">Arbitrum One</a> or when a DACert is posted by <a data-quicklook-from="arbitrum-nova">Arbitrum Nova</a>. This would be on the order of tens of minutes.
```